composer.json

```diff
@@ -16,7 +16,7 @@
     "minimum-stability": "stable",
     "require": {
         "composer-plugin-api": "^2.6",
-        "php-tuf/php-tuf": "0.1.5",
+        "php-tuf/php-tuf": "dev-main",
```

Comment:
Conditionally approving based on use not needing dev-main but then next php-tuf/php-tuf tag